pkgs/top-level/all-packages.nix

```diff
@@ -4583,7 +4583,7 @@ in
 
   isync = callPackage ../tools/networking/isync { };
 
-  itm-tools = callPackage ../development/tools/misc/itm-tools { };  
```

Comment:
There's a bunch of extraneous whitespace cleaning in this file.

Reply:
Seems to be the project's `.editorconfig` doing its thing.

https://github.com/NixOS/nixpkgs/blob/cbf2b8880b6fbe764cf40df7c0078536929e6cb6/.editorconfig#L11

I think it might be better to separate the commit and keep it as-is because other contributors who also have editorconfig enabled are also going to run into this anyways.

Reply:
I did a rebase to fix merge conflicts, and it seems the whitespace issues are now gone.